guarddog/cli.py

```diff
@@ -83,11 +91,17 @@ def _verify(path, rules, exclude_rules, output_format, exit_non_zero_on_finding,
 
     if output_format == "json":
         import json as js
-        print(js.dumps(results))
+        rv = js.dumps(results)
```

Comment:
not very clear what `rv` stands for

Reply:
it's a sqreenism 😅 , return_value, I should rename to that